next.config.js

```diff
@@ -1,3 +1,5 @@
+const removeImports = require('next-remove-imports')()
```

Comment:
I see the dependency says this is a requirement for tit to work with Next.js, which is not good. Basically this implies we have to double check every dependency we use which might import their own CSS. Data table component, toast, etc.

Adding this `next-remove-imports` means we remove all those imports from all dependencies which seems excessive to get 1 dependency to work

Reply:
Ok, I will check what are the consequences and see if it can be removed (or some other tweak).

Reply:
without `next-remove-imports`, this doesn't work at all. 😞